packages/@aws-cdk/toolkit-lib/lib/api/deployments/deploy-stack.ts

```diff
@@ -202,9 +206,31 @@ export interface DeployStackOptions {
 
 export async function deployStack(options: DeployStackOptions, ioHelper: IoHelper): Promise<DeployStackResult> {
   const stackArtifact = options.stack;
-
   const stackEnv = options.resolvedEnvironment;
 
+  let deploymentMethod = options.deploymentMethod ?? { method: 'change-set' };
+  // Honour old hotswap option
+  // @TODO remove when we don't care about this export anymore
```

Comment:
Why not remove now?

Reply:
This is API is currently exported as part of the legacy API exports from the CLI. I think we said we don't want to break them. Will update the comment to be more explicit.